src/chapterCleaner.test.ts

```diff
@@ -0,0 +1,325 @@
+// noinspection ALL
```

Comment:
Could you do a separate commit with the renaming and then the changes? Should be way easier to see the diff then.